jqColorPicker.js

```diff
@@ -272,7 +272,7 @@
             animationSpeed: 150,
             GPU: true,
             doRender: true,
-            customBG: '#FFF',
+            customBG: $(this).css('background-color') || '#FFF',
```

Comment:
required if options.customBG is empty, so the element background color doesn't change to '#FFF'